04-suspense-and-async-mode/exercise-fetch-solution/.eslintrc

```diff
@@ -0,0 +1,352 @@
+{
```

Comment:
I think having just one `.eslintrc` should be enough. If all are the same.

Reply:
Good tip, thank you. Unified in the commit 66979fd0a2dc91ddf3538fc8853206aa1b171b86